src/domain/app/App.tsx

```diff
@@ -56,6 +57,9 @@ class App extends React.Component<AppProps> {
           <PrivateRoute exact path={`/${locale}/registration/form`}>
             <RegistrationForm />
           </PrivateRoute>
+          <PrivateRoute exact path={`/${locale}/registration/welcome`}>
```

Comment:
`/form/success` ?

Reply:
form? why form? it's part of the registration process

Do you want that change because it feels right as a developer or because you think it is more intuitive for end users?

Reply:
now it is `registration/success`